strangerstudios/pmpro-payment-plans
# Fixes default level dropdown

### All Submissions:






### Changes proposed in this Pull Request:

Fixes an issue with the default plan dropdown not showing the stored value

### How to test the changes in this Pull Request:

1. Create multiple plans in a level
2. Set one of them to be the default seleted level
3. After saving, check if the dropdown reflects the correct value

### Other information:
